subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy

```diff
@@ -26,7 +25,7 @@ import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
```

Comment:
Did you mean to disable a bunch of smoke tests for 3rd-party plugins?

Reply:
nope, fixed